contents/builder-community-incubator.mdx

```diff
@@ -0,0 +1,140 @@
+      <ul className="m-0 p-0 list-none inline-grid sm:grid-cols-2 lg:grid-cols-3 gap-[10px] justify-evenly relative text-center sm:text-left">
+        <li className="bg-[#E5E7E0] dark:bg-[#2C2C2C] relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12 lg:px-8 rounded-2xl shadow-sm transition-colors duration-300">
+          <h5 className="text-xl font-extrabold m-0 pb-1 pr-4 text-black dark:text-white">$800 cash grant</h5>
+          <p className="m-0 text-[15px] text-gray-800 dark:text-gray-300">
+            Meant to get you going and to be used at organizer's discretion.
+          </p>
+        </li>
+        <li className="bg-[#E5E7E0] dark:bg-[#2C2C2C] relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12 lg:px-8 rounded-2xl shadow-sm transition-colors duration-300">
+          <h5 className="text-xl font-extrabold m-0 pb-1 pr-4 text-black dark:text-white">Help attracting attendees</h5>
+          <p className="m-0 text-[15px] text-gray-800 dark:text-gray-300">
+            We'll spread the word on your behalf if you need it.
+          </p>
+        </li>
+        <li className="bg-[#E5E7E0] dark:bg-[#2C2C2C] relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12 lg:px-8 rounded-2xl shadow-sm transition-colors duration-300">
+          <h5 className="text-xl font-extrabold m-0 pb-1 pr-4 text-black dark:text-white">Help finding a venue</h5>
+          <p className="m-0 text-[15px] text-gray-800 dark:text-gray-300">
+            The ideal location is a free space which we can source.
+          </p>
+        </li>
+      </ul>
+    </div>
+  </div>
+</div>
+
+<div className="grid md:grid-cols-2 gap-8 items-start my-12">
+  <div>
+    <h2 className="text-3xl lg:text-4xl mb-6">What we're asking from you, pilot participants</h2>
+    <p>
+      Bring together a group of at least 10 people to co-work and build for 5 straight weeks. The goal is to give you the initial push but let you carry forward afterwards.
+    </p>
+    <p>
+      Create a consistent time and structure that works for your group. For times, do evenings or weekends. For format, we suggest two ingredients: building and demo-ing.
+    </p>
+    <p>
+      Let us know how it's going along the way.
+    </p>
+  </div>
+  <div className="relative">
+    <div className="border-8 border-white shadow-xl rounded relative rotate-1">
+      <CloudinaryImage
+        src="https://res.cloudinary.com/dmukukwp6/image/upload/cambridge_9a4e27f42e"
+        alt="Builder community in action"
+        className="w-full"
+      />
+      <div className="absolute bottom-0 left-0 w-full text-white bg-gradient-to-b from-black/0 to-black/50 p-2 pt-8 text-sm">
+        A builder community taking over a coffee shop
+      </div>
+    </div>
+  </div>
+</div>
+
+<section id="apply" class="my-24 px-5">
+<div class="max-w-screen-md mx-auto">
+<h2 class="text-3xl lg:text-4xl text-center mb-8">Apply to lead a chapter</h2>
+<div class="bg-accent dark:bg-accent-dark p-8 rounded-lg">
+
+<iframe
```

Comment:
This sets a cookie which we probably don't want to do. Can we submit a form as props on a PostHog event and route it from there?